### ISSUE TITLE
PL docs needed 

### ISSUE THREAD

**Issue by trombonekenny** [eddelbuettel/pl-r-demos#5]
https://prairielearn.readthedocs.io/en/latest/ has sections ofr Python autograder, C/C++ autograder, Java autograder. We should make something similar for the R autograder

(I know that's not necessarily in this repo, but I'm thinking about how to merge this content into the PL core exampleCourse and docs.)

**Comment by eddelbuettel** [comment]
Yes, a good to do.  Have to figure out if that will be sourced from this repo or elsewhere.

**Comment by trombonekenny** [comment]
Up to you. If you're comfortable drafting in the PL PR, we could do it there. Or we could do it here and copy the file over when we're satisfied with the contents. In PL might be more direct.
